csharp/ql/test/resources/stubs/_frameworks/Microsoft.AspNetCore.App/Microsoft.AspNetCore.Components.Authorization.cs

```diff
@@ -14,61 +13,53 @@ public class AuthenticationState
                     public AuthenticationState(System.Security.Claims.ClaimsPrincipal user) => throw null;
                     public System.Security.Claims.ClaimsPrincipal User { get => throw null; }
                 }
-
                 public delegate void AuthenticationStateChangedHandler(System.Threading.Tasks.Task<Microsoft.AspNetCore.Components.Authorization.AuthenticationState> task);
-
                 public abstract class AuthenticationStateProvider
                 {
-                    public event Microsoft.AspNetCore.Components.Authorization.AuthenticationStateChangedHandler AuthenticationStateChanged;
+                    public event Microsoft.AspNetCore.Components.Authorization.AuthenticationStateChangedHandler AuthenticationStateChanged { add { } remove { } }
```

Comment:
Could be `add => throw null; remove => throw null;`. Or could be omitted altogether as before.

Reply:
I will remove them.